pkg/web/api/fromproto/connector.go

```diff
@@ -31,8 +31,11 @@ func ConnectorType(in apiv1.Connector_Type) connector.Type {
 }
 
 func ConnectorConfig(in *apiv1.Connector_Config) connector.Config {
-	return connector.Config{
-		Name:     in.Name,
-		Settings: in.Settings,
+	if in != nil {
```

Comment:
Nitpick: We generally strive to keep the "happy path" to the left (i.e. not indented) and return early in case of unexpected conditions. So I'd rather turn these statements around and check if `in == nil`.